internal/inbox/inbox_test.go

```
@@ -0,0 +1,106 @@
+	_, err = mgr.Create(makeInbox("Duplicate primary", "SUPPORT@example.com"))
+	require.Error(t, err)
+	_, err = mgr.Create(makeInbox("Alias hits primary", "sales@example.com", "support@example.com"))
+	require.Error(t, err)
+	_, err = mgr.Create(makeInbox("Alias hits alias", "help@example.com", "BILLING@example.com"))
+	require.Error(t, err)
+
+	updated, err := mgr.Update(first.ID, makeInbox("Support", "support@example.com", "accounts@example.com"))
+	require.NoError(t, err)
+	require.Equal(t, "accounts@example.com", updated.Aliases[0].Email)
+	var status string
+	_, err = db.Exec(`UPDATE inbox_email_addresses SET verification_status = 'verified' WHERE inbox_id = $1 AND email = 'accounts@example.com'`, first.ID)
```

golangci-lint (2.13.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)